docs/guide/upgrade/migrate_v1_v2.md

```diff
@@ -15,17 +15,11 @@ It supports existing AWS resources provisioned by AWSALBIngressController(>=v1.1
 
 1. The AWS LoadBalancer resource created for your Ingress will be preserved.
     
-2. If a numeric TargetPort is used in your service, the AWS TargetGroups created for your Ingress will be re-created.
-
-    !!!warning "downtimes"
-        This would cause downtimes to your service during targets registration into new TargetGroups created.
-    
-    !!!tip "details"
-        * The AWSALBIngressController always used `1` as TargetGroup's port.
-        * The AWSLoadBalancerController will use 
-            * the actual numeric TargetPort as TargetGroup's port if a numeric TargetPort used.
-            * `1` as TargetGroup's port if a lexical TargetPort used.
-        * The AWSLoadBalancerController will automatically create new TargetGroups and cleanup old TargetGroups if any.
+2. The AWSALBIngressController always used `1` as TargetGroup's port. The AWSLoadBalancerController will use 
+   * the actual numeric `targetPort` of the corresponding Kubernetes `Service` as TargetGroup's port if a numeric `targetPort` is used.
+   * `1` as TargetGroup's port if a lexical `targetPort` is used.
+   
+   Existing TargetGroups will be preserved; if a numerical `targetPort` is used, the TargetGroup's port will not be updated and stay `1`.
```

Comment:
You can put this line in the description for this list item 
```
2. Existing Targetgroups ...

      !!! tip ...
```

Reply:
I moved the
```
Existing TargetGroups will be preserved; if a numerical `targetPort` is used, the TargetGroup's port will not be updated and stay `1`
```
into a "details" block.

Reply:
Let me know if that's what you want.